project.mml

```diff
@@ -2101,20 +2100,16 @@ Layer:
           ('road', 330),
           ('living_street', 320),
           ('pedestrian', 310),
-          ('raceway', 300),
           ('motorway_link', 240),
           ('trunk_link', 230),
           ('primary_link', 220),
           ('secondary_link', 210),
           ('tertiary_link', 200),
-          ('service', 150),
-          ('cycleway', 140),
-          ('path', 130),
-          ('construction', 10)
+          ('service', 150)
         ) AS ordertable (highway, prio)
         USING (highway)
         WHERE highway IN ('motorway', 'motorway_link', 'trunk', 'trunk_link', 'primary', 'primary_link', 'secondary', 'secondary_link', 'tertiary',
-          'tertiary_link', 'residential', 'unclassified', 'road', 'service', 'pedestrian', 'raceway', 'living_street', 'construction', 'cycleway', 'path')
```

Comment:
With these edits, we lost the names of raceways, is it expected?

Reply:
Yes, I don’t see the point of having raceway names in a pov of a cyclist.